dataplane/ebpf/src/ingress/udp.rs

```diff
@@ -30,17 +30,16 @@ pub fn handle_udp_ingress(ctx: TcContext) -> Result<i32, i64> {
     };
 
     let backend = unsafe { BACKENDS.get(&key) }.ok_or(TC_ACT_PIPE)?;
-
-    let daddr_dot_dec = ip_from_int(unsafe { (*ip_hdr).daddr });
-    info!(
-        &ctx,
-        "Received a UDP packet destined for svc ip: {}.{}.{}.{} at port: {}",
-        daddr_dot_dec[0],
-        daddr_dot_dec[1],
-        daddr_dot_dec[2],
-        daddr_dot_dec[3],
-        u16::from_be(unsafe { (*udp_hdr).dest })
-    );
+    
+    match get_src_ip(&ctx) {
```

Comment:
@shaneutt  & @astoycos , not sure if this is idiomatic to use match here cause its still lot more line.
Any sugesstions ? or if this look I will refactor code from other part as well where old function was being

Reply:
so I _think_ you should just be able to do 

```
info!("Received a UDP packet destined for svc ip:: {:ipv4} at Port: {} ", unsafe { (*ip_hdr).daddr }, u16::from_be(unsafe {(*udp_hdr).dest}));
      
```

And avoid the `get_src_ip` function 
 
Also I think you can remove the `ip_from_int` as well

Reply:
Lol yeah this make sense no point in writing function

Reply:
I refactored this across udp, tcp and icmp. Also removed old function.
At same time i noticed while testing (could be another discussion altogether), I think tcp ep, doesnt route or not sure if its issue with my setup. I tried using main branch but same issue.

Reply:
TCP is only partially implemented at this time and is not fully working yet, so don't worry. #15 is tracking this, but is incomplete.

As long as UDP is still working (the integration tests check this) and you've verified the IP address display is still good then we should be OK :+1:

Reply:
Oh perfect thanks for clarifying @shaneutt